src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -2227,6 +2235,27 @@ bool GameObject::IsLootAllowedFor(Player const* player) const
     return true;
 }
 
+GameObject* GameObject::GetLinkedTrap()
+{
+    uint32 linkedEntry = GetGOInfo()->GetLinkedGameObjectEntry();
+
+    if (!linkedEntry)
+        return nullptr;
+
+    GameObject* linkedTrap = nullptr;
+
+    if (m_linkedTrap.Empty())
```

Comment:
We do. Master is spawned first, https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Spells/SpellEffects.cpp#L3533 - this code block summons linked trap (and you can link them together here)